docs/src/pages/guides/01-harry-potter.mdx

```diff
@@ -145,7 +145,7 @@ Response as Draco Malfoy: My favorite room in Hogwarts is the Slytherin Common R
 Finally, we'll demonstrate how to stream the response from the model. Streaming is useful for handling longer outputs or providing real-time feedback.
 
 ```typescript
-  const stream = await llm.stream({
+  const stream = await llm.__stream({
```

Comment:
this should be .stream

Reply:
Oh yeah I see we changed that. Making an update now.